src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UpdateRfocAcceptedStatus/UpdateRfocAcceptedCommandHandler.cs

```diff
@@ -159,20 +155,11 @@ private async Task<IList<string>> GetMcPkgNosToUpdateRfocStatusAsync(IList<strin
             return pcsMcPkgs.Where(mc => mc.OperationHandoverStatus == "ACCEPTED").Select(mc => mc.McPkgNo).ToList();
         }
 
-        private async Task<IList<string>> GetCommPkgNosToUpdateRfocStatusAsync(IList<string> commPkgNos, Project project)
-        {
-            var pcsCommPkgs = await _commPkgApiService.GetCommPkgsByCommPkgNosAsync(project.Plant, project.Name, commPkgNos);
-            return pcsCommPkgs.Where(c => c.OperationHandoverStatus == "ACCEPTED").Select(c => c.CommPkgNo).ToList();
-        }
-
-        private async Task<Certificate> GetOrCreateCertificateAsync(Guid certificateGuid, Project project, CancellationToken cancellationToken)
-            => await _certificateRepository.GetCertificateByGuid(certificateGuid) ?? await AddCertificateAsync(certificateGuid, project, cancellationToken);
 
-        private async Task<Certificate> AddCertificateAsync(Guid certificateGuid, Project project, CancellationToken cancellationToken)
+        private Certificate AddCertificate(Guid certificateGuid, Project project)
```

Comment:
dobbelchecking,  can never exist from before?

Reply:
Should not exist in the way we deal with certificates in IPO today. We create certificate instance if we receive an event with RFOC accepted. This event should only be sent once. If it is sent more than once something is wrong somewhere else (dublicate events). Should I handle this?

Reply:
Should be fine. as long as there is unique constraint on certificateGuid. Then it will fail, and thats good :)